lineflow/simulation/line.py

```diff
@@ -174,47 +175,51 @@ def _register_objects_at_env(self):
         for o in self._objects.values():
             o.register(self.env)
 
-    def _draw(self, screen, actions=None):
+    def _draw(self, actions=None):
```

Comment:
I moved more stuff into the `Viewpoint` class. Maybe - after the refactoring - all pygame specific things can happen inside `Viewpoint`?